src/frontend/test_runner/tests/plan_test_runner.rs

```diff
@@ -12,6 +12,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Planner test runner, auto generated
+use std::ffi::OsStr;
 
-include!("gen/testcases.rs");
+use libtest_mimic::{run_tests, Arguments, Outcome, Test};
+use risingwave_frontend_test_runner::run_test_file;
+use tokio::runtime::Runtime;
+use walkdir::WalkDir;
+
+fn main() {
+    let run_tests_args = &Arguments::from_args();
+    let mut tests = vec![];
+
+    for entry in WalkDir::new("tests/testdata") {
+        let entry = entry.unwrap();
+        let path = entry.path();
+        if !path.is_file() {
+            continue;
+        }
+
+        if (path.extension() == Some(OsStr::new("yml"))
+            || path.extension() == Some(OsStr::new("yaml")))
+            && !path
+                .file_name()
+                .unwrap()
+                .to_string_lossy()
+                .contains(".apply.yaml")
+        {
+            let file_name = path.file_name().unwrap().to_string_lossy();
+            let test_case_name = file_name.split('.').next().unwrap();
+            tests.push(Test {
+                name: format!("{test_case_name}_test"),
+                kind: "".into(),
+                is_ignored: false,
+                is_bench: false,
+                data: (test_case_name.to_string(), file_name.to_string()),
+            });
+        }
+    }
+
+    if tests.is_empty() {
+        panic!("no test case found in planner test!");
+    }
+
+    run_tests(run_tests_args, tests, |test| {
+        let (test_case_name, file_name): (String, String) = test.clone().data;
+        let path = std::path::Path::new(env!("CARGO_MANIFEST_DIR"))
+            .join("tests")
+            .join("testdata")
+            .join(file_name);
+
+        let file_content = Runtime::new()
```

Comment:
For file read, we can simply use `std::fs::read`

Reply:
thx. fixed